tools/checkcerts.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python3
+
+import argparse
+import socket
+import ssl
+import subprocess
+import sys
+import os
+import tempfile
+import datetime
+import smtplib
+
+DAYS_BEFORE_WARN=3
```

Comment:
Should we make this a bit longer? We're in trouble if a cert wants to expire at the end of a long weekend.

Reply:
I have no problem with that

Reply:
Updated to 7 days